packages-backend/loggers/src/middlewares/create-access-logger.ts

```diff
@@ -39,15 +39,20 @@ const createAccessLoggerMiddleware = (
     skip: (req) =>
       Boolean(options.silent) || ignoreUrls.includes(req.originalUrl),
     dynamicMeta: (req) => {
-      const remoteAddress = req.socket?.remoteAddress;
-      const proxyIps = parseIps(req);
-      const [clientIp] = proxyIps;
-      return {
-        clientIp: clientIp ?? remoteAddress,
-        proxyIps,
-        hostname: req.hostname,
-        ...(remoteAddress ? { remoteAddress } : {}),
-      };
+      try {
+        const remoteAddress = req.socket?.remoteAddress;
```

Comment:
Can we check the `req.socket` is not null first? I think the issue we got it because the socket isn't/wasn't success then there will be no remoteAddress as the result.

Reply:
Yes the `?.` does the null check